### ISSUE TITLE
Front-end bug with CPT + term_meta + rich_text (Google Chrome)

### ISSUE THREAD

**Issue by dmhendricks** [htmlburger/carbon-fields#365]
I'm not sure of something else is causing this, but I have a custom taxonomy named "Categories" for a custom post type that I created.

I added two `rich_text` fields to the taxonomy term fields.

```
Container::make( 'term_meta', __( 'Term Options', 'crb' ) )
  ->where( 'term_taxonomy', '=', 'my_cpt_category' )
  ->add_fields( array(
    Field::make( 'rich_text', 'my_cpt_category_description', __( 'Description', 'somedomain' ) )
      ->set_rows( 8 ),
    Field::make( 'rich_text', 'my_cpt_category_footer', __( 'Category Footer Text', 'somedomain' ) )
      ->set_rows( 8 )
  )
);
```

When I visit the taxonomy page and then refresh or click _any_ link, it gives me the "Do you want to leave this site? Changes you made may not be saved." popup. I have changed nothing, and it happens by simply visiting the taxonomy editor page and then immediately hitting refresh (I am not hitting any other keys - I have tried it multiple times).

Any idea if this is something with my setup, or maybe a bug?

Crude illustration (pardon the relative size of the JS popup - it was quick & dirty):

![carbon-fields-term_meta-js-dialog](https://user-images.githubusercontent.com/8184361/31029250-a7f4a582-a516-11e7-9f32-38c211ffa599.png)

- PHP 7.0.3
- WordPress 4.8.2
- Carbon Fields 2.1.0 (but if I remember correctly, I think that it was happening with 2.0.4 as well)
- Mac OS X Sierra
- Chrome 60

**Update:** This does not appear to happen on Firefox 55 for me - only Chrome. When I load the page in FF, the focus is set the the first field (Name), but even if I change the focus to the rich_text field, I can refresh and/or click a link without the popup.

**Update 2:** Curiously, I just tried the page in [Vivaldi](https://vivaldi.com) (which uses Blink for rendering, as Chrome) and I don't get the popup. I have not tried it on Windows/Edge yet, but I did confirm that another user (also OS X Sierra) is getting the same behavior in Chrome. I will try on Edge (as well as a clean install using the same plugin) on a clean install later tonight.

Thank you,
Daniel

**Comment by atanas-angelov-dev** [comment]
Hi @dmhendricks ,

We have a fix for this in our `development` branch (5e3399b470e9ea8bb82b14fdd669e315a58b3cc4). The fix will make it into the next patch version - I will close this issue when it's live.

**Comment by dmhendricks** [comment]
Ok, thank you A.

**Comment by atanas-angelov-dev** [comment]
Fixed in 2.1.1 - closing.
